### ISSUE TITLE
Broken tests

### ISSUE THREAD

**Issue by Nikash** [ember-polyfills/ember-getowner-polyfill#12]
Latest changes to the polyfill seem to break our integration tests which use `liquid-fire`.

Not able to exactly pin-point the change that is causing the failure. When testing a component which uses the `liquid-if`, the call to `this.render` throws the following error 

>TypeError: '[object Object]' is not a function (evaluating '(0, _emberGetownerPolyfill["default"])(this)')

If we add a direct dependency on `v.1.0.0` of the polyfill our tests pass. I'm guessing some adaptation to either `liquid-fire` or the polyfill itself is required.

@ef4 @rwjblue could you please let me know what you think?

**Comment by ef4** [comment]
Do you have RAISE_ON_DEPRECATION=true? 

If so, this is being caused by https://github.com/rwjblue/ember-getowner-polyfill/pull/11.

It's just hard to see due to https://github.com/ember-cli/loader.js/pull/90, which has been annoying me for a long time. I just went over there are merged that.

**Comment by Nikash** [comment]
Nope, don't have `RAISE_ON_DEPRECATION=true`.

Once the fix in `loader.js` is in `ember-cli`'s default blueprint, will it help with my debugging?

**Comment by ef4** [comment]
I think it will. loader.js 4.1.0 is out now, you can try updating just that and see if you get a better error message.

**Comment by Nikash** [comment]
I wasn't really able to test with loader.js 4.1.0 because I ran into [this](https://github.com/ember-cli/loader.js/issues/66) issue.

But something else I noticed which could be useful is that, we are on `ember 1.13.12`. But `liquid-fire` seems to use `ember 2.9.0`. Because of the ['deprecation' notice addition](https://github.com/rwjblue/ember-getowner-polyfill/commit/51d545c542ee30aad69d018a94c9f2b3fc121aeb#diff-7562eb8eba3c3d86a671045f8c70107e), the polyfill does not seem to be loaded causing the [`getOwner`](https://github.com/ember-animation/liquid-fire/blob/58a034a5a01fd873cd983bfa21eb39ff100ccbeb/addon/transition-map.js#L14) call to fail with the error I mentioned. I think this is the case, because I see the deprecation warning on my browser when running the test followed by the error ` _emberGetownerPolyfill.default is not a function(…)`.
